src/thing_objects.c

```diff
@@ -721,9 +721,9 @@ static long food_moves(struct Thing *objtng)
     TbBool dirct_ctrl = is_thing_directly_controlled(objtng);
     if (dirct_ctrl)
     {
-        if (objtng->food.byte_16 > 0)
+        if (objtng->food.possession_freeze_timer > 0)
```

Comment:
freeze would make sense if we would not have a spell to _actually_ freeze units.

Reply:
named it possession_startup_timer